Strategies/TransitionToPartner.md

```diff
@@ -82,26 +82,26 @@ This means build automation, test automation, continuous integration, release au
 
 &#042; Architecture compliance automation needs to be further evaluated.
 
-## 5. Shift to Product Management
+## 5. Shift to Agile Project Management
 
 IITB will adopt a product management mindset wherein each Solution is viewed as a product and managed throughout its entire lifespan.
 
 Solutions will no longer be delivered as projects that culminate with a release, but rather as products that are continuously improved through smaller, more frequent releases.
```

Comment:
We'll need to change that sentence too. I'd say
Solutions will be delivered via agile projects, which make use of products that are continuously improved through smaller, more frequent releases.